tests/unit/test_s3_deltalake.py

```diff
@@ -1,22 +1,66 @@
 from __future__ import annotations
 
-from typing import Any
+from typing import Any, Iterator
 
+import boto3
 import pytest
 
 import awswrangler as wr
 import awswrangler.pandas as pd
 
+from .._utils import (
+    get_time_str_with_random_suffix,
+)
+
+
+@pytest.fixture(scope="session")
+def lock_dynamodb_table() -> Iterator[str]:
```

Comment:
Any reason why you have a different fixture for creating a dynamodb table to the [existing](https://github.com/aws/aws-sdk-pandas/blob/main/tests/conftest.py#L219) one?

Reply:
Mainly because I wanted the scope to be `session` rather than `test`.